docs/iris/src/further_topics/metadata.rst

```diff
@@ -0,0 +1,987 @@
+:class:`~iris.common.metadata.CubeMetadata`,
+
+.. doctest:: metadata-convert
+
+    >>> cube.metadata  # doctest: +SKIP
+    CubeMetadata(standard_name='air_temperature', long_name=None, var_name='air_temperature', units=Unit('K'), attributes={'Conventions': 'CF-1.5', 'STASH': STASH(model=1, section=3, item=236), 'Model scenario': 'A1B', 'source': 'Data from Met Office Unified Model 6.05'}, cell_methods=(CellMethod(method='mean', coord_names=('time',), intervals=('6 hour',), comments=()),))
+
+We can easily convert it to a :class:`~iris.common.metadata.DimCoordMetadata` instance
+using ``from_metadata``,
+
+.. doctest:: metadata-convert
+
+    >>> DimCoordMetadata.from_metadata(cube.metadata)  # doctest: +SKIP
+    DimCoordMetadata(standard_name='air_temperature', long_name=None, var_name='air_temperature', units=Unit('K'), attributes={'Conventions': 'CF-1.5', 'STASH': STASH(model=1, section=3, item=236), 'Model scenario': 'A1B', 'source': 'Data from Met Office Unified Model 6.05'}, coord_system=None, climatological=None, circular=None)
+
+By examining :numref:`metadata members table`, we can see that the
+:class:`~iris.cube.Cube` and :class:`~iris.coords.DimCoord` container
+classes share the following common metadata members,
+
+- ``standard_name``,
+- ``long_name``,
+- ``var_name``,
+- ``units``, and
+- ``attributes``
+
+As such, all of these metadata members of the
+:class:`~iris.common.metadata.DimCoordMetadata` instance are populated from the associated
+:class:`~iris.common.metadata.CubeMetadata` instance members. However, a
+:class:`~iris.common.metadata.CubeMetadata` class does not contain the following
+:class:`~iris.common.metadata.DimCoordMetadata` members,
+
+- ``coords_system``
+- ``climatological``, and
+- ``circular``
+
+Thus these particular metadata members are set to ``None`` in the resultant
+:class:`~iris.common.metadata.DimCoordMetadata` instance.
+
+Note that, the ``from_metadata`` method is also available on a metadata
+class instance,
+
+.. doctest:: metadata-convert
+
+    >>> longitude.metadata.from_metadata(cube.metadata)
+    DimCoordMetadata(standard_name='air_temperature', long_name=None, var_name='air_temperature', units=Unit('K'), attributes={'Conventions': 'CF-1.5', 'STASH': STASH(model=1, section=3, item=236), 'Model scenario': 'A1B', 'source': 'Data from Met Office Unified Model 6.05'}, coord_system=None, climatological=None, circular=None)
+
+
+Metadata assignment
+^^^^^^^^^^^^^^^^^^^
+
+.. testsetup:: metadata-assign
+
+   import iris
+   cube = iris.load_cube(iris.sample_data_path("A1B_north_america.nc"))
+   longitude = cube.coord("longitude")
+   original = longitude.copy()
+   latitude = cube.coord("latitude")
+
+The ``metadata`` property available on each Iris `CF Conventions`_ container
+class (see :numref:`metadata classes table`) can be use not only **to get**
+the metadata of an instance, but also **to set** the metadata on an instance.
+
```

Comment:
For some reason I did not pickup on the difference between longitude and latitude.  Maybe just one line and one box would be ok:

```
For example, we can assign the :class:`~iris.common.metadata.DimCoordMetadata` metadata from the
``longitude`` coordinate directly to the ``latitude`` coordinate,

.. doctest:: metadata-assign

    >>> longitude.metadata
    DimCoordMetadata(standard_name='longitude', long_name=None, var_name='longitude', units=Unit('degrees'), attributes={}, coord_system=GeogCS(6371229.0), climatological=False, circular=False)
    >>> longitude.metadata = latitude.metadata
    >>> longitude.metadata
    DimCoordMetadata(standard_name='latitude', long_name=None, var_name='latitude', units=Unit('degrees'), attributes={}, coord_system=GeogCS(6371229.0), climatological=False, circular=False)
```

Reply:
@abooton I've printed the `latitude.metadata` just to clarigy, what's what...